### PR TITLE
Changed: Sugestões de melhorias no readme.md

### PR DESCRIPTION
**Pull request by eudiegoborgs**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -57,9 +58,9 @@ Requisitos:
 
 ### Payload
 
-Faça uma **proposta** de payload, mas trazemos um exemplo aqui:
+Nós iremos rodar validar seu projeto com um teste automatizado, por isso é importante que você mantenha esse payload em suas requisições:
```
Comment (lucascavalcante) [inline readme.md]: Fala Diego! Faltou um "e" entre as palavras "rodar" e "validar".

**Review by lucascavalcante — APPROVED** [review]

**Review by rafael-neris — APPROVED** [review]

**Review by fabiohsmachado — APPROVED** [review]

File: readme.md [hunk 2]
```diff
@@ -69,6 +70,14 @@ POST /transaction
 }
 ```
 
+Response: POST /transaction
+```json
+{
+    "status" : "ok" || "error",
```
Comment (josemiguelq) [inline readme.md:84]: isso podia ir no http code, o q acha?

**Review by josemiguelq — APPROVED** [review]

**Comment by RicardoZandonaiPicPay** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [eaa4c64a\-6214\-4de6\-9494\-863acf7d1f52](https://ast.checkmarx.net/projects/90b64d2f-4e5b-44ad-bdc0-ea62b5347abe/scans?id=eaa4c64a-6214-4de6-9494-863acf7d1f52&branch=master)
### No New Or Fixed Issues Found

**Comment by leocavalcante** [comment]
Stale + supersede: https://github.com/PicPay/picpay-desafio-backend/pull/42

**Pull request closed without merging**